core/api/src/main/java/com/blazebit/notify/NotificationProcessor.java

```diff
@@ -39,5 +39,5 @@ default Object process(N jobInstance, JobInstanceProcessingContext<Object> conte
      * @param notification The notification to process
      * @param context The job instance context
      */
-    void process(N notification, NotificationJobContext context);
+    void processWithoutResult(N notification, JobInstanceProcessingContext<Object> context);
```

Comment:
I don't mind the method rename, but please restore the argument type signature. I don't want implementers to have to do the cast.

Reply:
The thing is, I need the `JobInstanceProcessingContext` to invoke `com.blazebit.job.JobInstance#markDone` in `com/blazebit/notify/impl/NotificationProcessorImpl.java:66`. The signature of `com.blazebit.job.JobInstance#markDone` was changed in https://github.com/Blazebit/blaze-job/commit/981d99856532c9169544118a3491ca151f78bdcb.